flow360/component/geometry.py

```diff
@@ -164,11 +110,14 @@ def file_names(self) -> List[str]:
 
     # pylint: disable=protected-access
     # pylint: disable=duplicate-code
-    def submit(self, progress_callback=None) -> Geometry:
-        """submit geometry to cloud
+    def submit(self, solver_version, description="", progress_callback=None) -> Geometry:
+        """
+        Submit geometry to cloud and create a new project
```

Comment:
why solver version and description goes here instead of constructor?

Reply:
solver_version is not a property / does not belong to the `GeometryDraft`. There is no reason to tie the "" with `GeometryDraft`.
`solver_version` is only a run time parameter for API calls.
The user can use the same GeometryDraft with different solver version.

Reply:
Not sure if I agree here. Solver version is provided for CaseDraft and it makes sense, as params depends on solver version. Later, we can hide solver version from user and only depend on python package version. But I would keep it consistent.

Reply:
Sure. Let me change it. I have no strong preference here.